src/FRP/Event.purs

```diff
@@ -103,6 +104,11 @@ foreign import sampleOn :: forall a b. Event a -> Event (a -> b) -> Event b
 sampleOn_ :: forall a b. Event a -> Event b -> Event a
 sampleOn_ a b = sampleOn a (b $> id)
 
+-- | Return only the events from the second stream that occur while the first
+-- | stream be true.
```

Comment:
Haha, it's actually subjunctive mood, but I won't subject everyone else to my grammar obsession... :flushed: